### PR TITLE
Fix:#91 Improve introductory section

### PR DESCRIPTION
**Pull request by smitrajput**
Improve the 'What is this' section to include a meaningful crux of the core features, the CLI offers.
Note: It has a dummy-logo, which is a mere placeholder/suggestion to the protofire team
## Cute animal picture
:poodle:

### CODE REVIEW

**Comment by fvictorio** [comment]
Thanks @smitrajput, this looks good! Don't worry about the CI, that's caused by the format of the commit messages. When this is ready to be merged I'm going to squash it and change the message.

@pablofullana could you also take a look?

**Comment by ChrisChinchilla** [comment]
@smitrajput You don't really need to use HTML in markdown, you especially don't need to use heading tags, or paragraph tags, better to use markdown syntax instead.

File: README.md [hunk 1]
```diff
@@ -58,6 +65,19 @@ section or the [list of commands](#commands) to find out what you can do with it
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Installation
+
+Install it globally:
```
Comment (ChrisChinchilla) [inline README.md]:
```suggestion
Install es-cli globally:
```

File: README.md [hunk 2]
```diff
@@ -58,6 +65,19 @@ section or the [list of commands](#commands) to find out what you can do with it
...
+
+`npm install -g eth-cli`
+
+You can also try it out with `npx`:
```
Comment (ChrisChinchilla) [inline README.md]:
```suggestion
You can also try es-cli out with `npx`:
```

**Comment by ChrisChinchilla** [comment]
Hey @smitrajput I've been working with these bounties to help polish before merging, just a couple of small things and we can get it merged and paid out.

**Comment by smitrajput** [comment]
> @smitrajput You don't really need to use HTML in markdown, you especially don't need to use heading tags, or paragraph tags, better to use markdown syntax instead.

I've got rid of the h1 and the 'p' tags, but I had to use the <center> tag instead, is that OK? Coz this is the only way I found other than html's, to center align texts. Or if you have a better alternative in mind to center align texts, plz let me know

**Comment by ChrisChinchilla** [comment]
@smitrajput Why does it need to be centred? This is just a read me file, most just use the default markdown syntax.

**Comment by smitrajput** [comment]
> @smitrajput Why does it need to be centred? This is just a read me file, most just use the default markdown syntax.

@ChrisChinchilla , just to make it look better. If the left alignment of the heading 'eth-cli' is better for you, then I think I should keep it left-aligned. Opinions?

**Comment by ChrisChinchilla** [comment]
@smitrajput I'd say just stick to vanilla markdown, GitHub readmes are typically pretty plain, and the content is the most important. :)

File: README.md (hunk 1 above)
Comment (fvictorio) [reply README.md]: Just in case, it's `eth-cli`

**Comment by smitrajput** [comment]
> @smitrajput I'd say just stick to vanilla markdown, GitHub readmes are typically pretty plain, and the content is the most important. :)

Made the following suggested changes:
- removed all HTML tags
- removed the logo, as it looked a bit odd, with the left-aligned texts
- replaced 'it' with 'eth-cli' wherever suggested
Please have a look

**Comment by fvictorio** [comment]
@smitrajput When everything is ready, please squash everything into a single commit with a message like `chore: improve readme introduction`. I can do it for you, but in that case I will appear as the committer, and I prefer that credit is properly given :slightly_smiling_face:

**Comment by smitrajput** [comment]
@ChrisChinchilla please have a look at my last commit

File: README.md [hunk 3]
```diff
@@ -58,6 +59,19 @@ section or the [list of commands](#commands) to find out what you can do with it
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Installation
+
+Install eth-cli globally:
+
+`npm install -g eth-cli`
```
Comment (ChrisChinchilla) [inline README.md]:
```suggestion
```shell
npm install -g eth-cli
```
```

**Comment by ChrisChinchilla** [comment]
@smitrajput One tiny suggestion and we're good to go @fvictorio

**Comment by smitrajput** [comment]
@ChrisChinchilla please have a look now

**Comment by fvictorio** [comment]
@smitrajput can you squash everything into a single commit with a message like `chore: improve intro in readme`?

**Comment by smitrajput** [comment]
> @smitrajput can you squash everything into a single commit with a message like `chore: improve intro in readme`?

@fvictorio I think I have squashed it already into a single commit. You can verify this by checking out the 'Commits' tab in this pull request, which shows '1 commit', namely '9aa1ef1'

**Comment by fvictorio** [comment]
Yes, that worked. Thanks @smitrajput!

**Comment by smitrajput** [comment]
@fvictorio I haven't yet received the bounty :sweat_smile:

**Comment by ChrisChinchilla** [comment]
@smitrajput @fvictorio doesn't have it, we will ow process it, but they are in an America time zone.

**Comment by smitrajput** [comment]
> @smitrajput @fvictorio doesn't have it, we will ow process it, but they are in an America time zone.

Received. Thanks!

**Pull request merged**